ldap/servers/slapd/back-ldbm/ldbm_modrdn.c

```diff
@@ -1279,11 +1281,6 @@ ldbm_back_modrdn(Slapi_PBlock *pb)
     goto common_return;
 
 error_return:
-    /* Revert the caches if this is the parent operation */
-    if (parent_op && betxn_callback_fails) {
-        revert_cache(inst, &parent_time);
```

Comment:
The removal of revert cache is worrisome.
 Could we really afford not restoring the modified entries (typically those modified by the plugin) ?
Same comment also apply for next change